guides/source/caching_with_rails.md

```diff
@@ -199,17 +190,92 @@ render(partial: "hotels/hotel", collection: @hotels, formats: :html, cached: tru
 
 Will load a file named `hotels/hotel.html.erb` in any file MIME type, for example you could include this partial in a JavaScript file.
 
-### Managing Dependencies
+### Low-Level Caching using `Rails.cache.fetch`
```

Comment:
I didn't change any content in the Low-Level Caching section, I just renamed it to Low-Level Caching using `Rails.cache.fetch` to be more explicit.